website/docs/components/copy/snippet/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,55 @@
+## Usage
+
+### When to use
+
+- To allow users to copy code snippets, such as URLs, tokens, IDs, etc.
+
+### When not to use
+
+- To copy content outside its container, use [Copy Button](/components/copy/button).
+- In a code block, use [Copy Button](/components/copy/button).
+
+### Effective Usage
+
+- There’s no limit to the number of `Copy::Snippet` components on a page, but maintain a balance to avoid overwhelming the user or cluttering the interface.
```

Comment:
When speaking of the Ember component, I think saying "components" above is fine since there isn't a concept of a component "instance" in code like there is in Figma.

Reply:
Yep that's a good point!